jax/_src/custom_derivatives.py

```diff
@@ -831,18 +830,21 @@ def rev(objective_fn, res, g):
   else:
     return _closure_convert_for_avals(fun, in_tree, in_avals)
 
+def _is_perturbed(x: Any) -> bool:
+  if isinstance(x, (pe.JaxprTracer, ad.JVPTracer)):
```

Comment:
```suggestion
  if isinstance(x, ad.JVPTracer):
```

If the tests pass with just this, it's... cleaner.